.env

```diff
@@ -132,7 +132,14 @@ CACHE_INDEFINITELY=false
 # CACHE_PREFIX must not contain colon symbol
 CACHE_PREFIX=local-chain
 # WHITELISTED_HEADERS contains comma-separated list of headers which has to be cached along with EVM JSON-RPC response
-WHITELISTED_HEADERS=Server,Vary,Access-Control-Expose-Headers,Access-Control-Allow-Origin,Access-Control-Allow-Methods,Access-Control-Allow-Headers,Access-Control-Allow-Credentials,Access-Control-Max-Age
+WHITELISTED_HEADERS=Vary,Access-Control-Expose-Headers,Access-Control-Allow-Origin,Access-Control-Allow-Methods,Access-Control-Allow-Headers,Access-Control-Allow-Credentials,Access-Control-Max-Age
+# DEFAULT_ACCESS_CONTROL_ALLOW_ORIGIN_VALUE contains default value for Access-Control-Allow-Origin header.
+# NOTE: it will be used only in Cache Hit scenario.
+DEFAULT_ACCESS_CONTROL_ALLOW_ORIGIN_VALUE="*"
+# Map contains mapping between hostname (for ex. evm.kava.io) and corresponding value for Access-Control-Allow-Origin header.
+# If hostname for specific request is missing we fallback to DEFAULT_ACCESS_CONTROL_ALLOW_ORIGIN_VALUE.
+# NOTE: it will be used only in Cache Hit scenario.
```

Comment:
please update caching docs with details on how the cachng middleware affects headers and logic for deciding when to modify cors headers